### PR TITLE
pkp/pkp-lib#6140 Displaying Upgrade section at OJS installation within a notification component

### PR DESCRIPTION
**Pull request by henriqueramos**
# What was made?

Added the upgrade section at OJS installation within a notification component.

# Evidences
## Current Version
103529433-6e59b100-4e64-11eb-8d93-a3e0266b2d33

## Updated Version
Screenshot from 2021-01-11 21-48-32

### CODE REVIEW

File: locale/en_US/locale.po [hunk 1]
```diff
@@ -1715,16 +1715,18 @@ msgstr "OJS Installation"
 msgid "installer.ojsUpgrade"
 msgstr "OJS Upgrade"
 
+msgid "installer.updatingInstructions"
+msgstr ""
+"\n"
+"<notification>\nIf you are upgrading an existing installation of OJS, <pkp-button element=\"a\" href=\"{$upgradeUrl}\">click here</pkp-button> to proceed.</notification>\n"
```
Comment (NateWr) [inline locale/en_US/locale.po:1721]: Let's move the `<notification>` tag out of the translation code. That can go directly into here: https://github.com/pkp/pkp-lib/pull/6571/files#diff-68710e4a718cc0c14dbdb1cfe373f032481bd8d26d42f08d6851499e3bcd860eR18.

Also, let's change `<pkp-button>` to the `<a>` element. I think the link will be prominent enough once the whole notice is in the `<notification>` element.

Comment (NateWr) [reply locale/en_US/locale.po:1721]: (The line breaks can also be removed.)

**Review by NateWr — CHANGES_REQUESTED** [review]
Great! Just the one comment on the localized string.

**Comment by henriqueramos** [comment]
# Updated version

Screenshot from 2021-01-13 02-15-19

**Pull request merged**